src/utils/utils.ts

```diff
@@ -31,3 +33,25 @@ export async function executeCommand(command: string, cwd?: string): Promise<str
 export function generateQuotedFullName(schema: string, objectName: string): string {
 	return `[${escapeClosingBrackets(schema)}].[${escapeClosingBrackets(objectName)}]`;
 }
+
+
+/**
+ * Gets a unique file name
+ * Increment the file name by adding 1 to function name if the file already exists
+ * @param folderPath selected azure project folder path
+ * @param fileName objectName that was chosen by the user
+ * @returns the function name that will always add at least `1` to the function name
+ */
+export async function getUniqueFileName(folderPath: string, fileName: string): Promise<string | undefined> {
+	let count: number = 0;
+	const maxCount: number = 1024;
```

Comment:
Why is 1024 chosen as the maxCount?

Reply:
This was just the default number. Azure Functions does the same thing with their Create Function.